account_journal_lock_date/models/account_move.py

```diff
@@ -15,18 +16,17 @@ class AccountMove(models.Model):
     @api.model
     def create(self, values):
         move = super().create(values)
-        move._check_lock_date()
+        move._check_fiscalyear_lock_date()
```

Comment:
Is it needed to call explicitly this method? Isn't done by standard?

Reply:
In the create of account.move, that method is not called, but it is called from the create method of account.move.line, so I think we could remove the override of the create method.

And in the override of the write method I left only one _check_fiscalyear_lock_date call